specs/RasterizationScale.md

```diff
@@ -0,0 +1,154 @@
+    m_webviewAdditionalRasterizationScale = additionalScale;
+    double rasterizationScale =
+        additionalScale * m_appWindow->GetDpiScale() * m_appWindow->GetTextScale();
+    CHECK_FAILURE(m_controller->put_RasterizationScale(rasterizationScale));
+}
+
+Callback<ICoreWebView2RasterizationScaleChangedEventHandler>(
+    [this](ICoreWebView2Controller* sender, IUnknown* args) -> HRESULT {
+        double rasterizationScale;
+        CHECK_FAILURE(m_controller->get_RasterizationScale(&rasterizationScale));
+
+        std::wstring message = L"WebView2APISample (RasterizationScale: " +
+            std::to_wstring(int(rasterizationScale * 100)) + L"%)";
+        SetWindowText(m_appWindow->GetMainWindow(), message.c_str());
+        return S_OK;
+    })
+.Get(), &m_rasterizationScaleChangedToken));
+```
+## .Net
+```c#
+public void SetRasterizationScale(double rasterizaitonScale)
+{
+    CoreWebView2Controller.ShouldDetectMonitorScaleChanges = false;
+    CoreWebView2Controller.RasterizationScale = rasterizationScale;
+}
+```
+
+
+# API Notes
+See [API Details](#api-details) section below for API reference.
+
+# API Details
+## Win32 C++
+```c#
+interface ICoreWebView2Controller2 : ICoreWebView2Controller {
+  /// The rasterization scale for the WebView. The rasterization scale is the
+  /// combination of the monitor DPI scale and text scaling set by the user.
+  /// This value should be updated when the DPI scale of the app's top level
+  /// window changes (i.e. monitor DPI scale changes or window changes monitor)
+  /// or when the text scale factor of the system changes.
+  ///
+  /// \snippet AppWindow.cpp DPIChanged
+  ///
+  /// \snippet AppWindow.cpp TextScaleChanged1
+  ///
+  /// \snippet AppWindow.cpp TextScaleChanged2
+  ///
+  /// Rasterization scale applies to the WebView content, as well as
+  /// popups, context menus, scroll bars, etc. Normal app scaling scenarios
+  /// should use the ZoomFactor property or SetBoundsAndZoomFactor API.
+  ///
+  /// \snippet ViewComponent.cpp RasterizationScale
+  [propget] HRESULT RasterizationScale([out, retval] double* scale);
+  // Set the rasteriation scale property.
+  [propput] HRESULT RasterizationScale([in] double scale);
+
+  /// ShouldDetectMonitorScaleChanges property determines whether the WebView
+  /// attempts to track monitor DPI scale changes. When true, the WebView will
+  /// track monitor DPI scale changes, update the RasterizationScale property,
+  /// and raises RasterizationScaleChanged event. When false, the WebView will
+  /// not track monitor DPI scale changes, and the app must update the
+  /// RasterizationScale property itself. RasterizationScaleChanged event will
+  /// never raise when ShouldDetectMonitorScaleChanges is false.
+  [propget] HRESULT ShouldDetectMonitorScaleChanges([out, retval] BOOL* value);
+  /// Set the ShouldDetectMonitorScaleChanges property.
+  [propput] HRESULT ShouldDetectMonitorScaleChanges([in] BOOL value);
```

Comment:
What happens when you modify RasterizationScale and ShouldDetectMonitorScaleChanges is true?

Reply:
The current implementation in experimental, is that it essentially does nothing. If the app sets the RasterizationScale while ShouldDetectMonitorScaleChanges is true, the WebView changes the RasterizationScale back to match the monitor scale and raises the RasterizationScaleChanged event.

We could change this behavior so that the RasterizationScale set by the app lasts until the monitor scales.  Or do you think the behavior is okay and just update the documentation to describe this behavior?

Reply:
I don't think its a scenario we need to support and current behavior sounds fine. Just document it.